keyboards/pdxkbc/keymaps/via/keymap.c

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2019 Franklin Harding
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    PDXKBCREDDIT = SAFE_RANGE,
+    PDXKBCDISCORD,
+    BADGELIFE,
+    HACKTHEPLANET
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+      PDXKBCREDDIT, PDXKBCDISCORD,
+      BADGELIFE,    HACKTHEPLANET,
+      KC_VOLU,      KC_VOLD
```

Comment:
```suggestion
      MACRO00, MACRO01,
      MACRO02, MACRO03,
      KC_VOLU,  KC_VOLD
```

Reply:
I could just remove the macros entirely as its very simple to add them back in via...VIA. 

Does Macro0, Macro01 still count as custom keycodes?

Reply:
These are the via specific ones <https://github.com/qmk/qmk_firmware/blob/master/quantum/via.h#L102> which load fine.

Reply:
@zvecr yeah, for the macropads I've done, I just put something simple and easy for keytesters to show it's working (i.e. numbers or F1-F6, etc.), since it's easy to set up exactly what you want with VIA and the default keymap isn't really important. Just wanted to avoid VIA showing the hex codes for these unknown user keycodes.